### PR TITLE
Looping Samples

### PR DESCRIPTION
**Pull request by ScottCarda-MS**
Adds samples for describing looping syntax in Q#.

### CODE REVIEW

File: samples/language/RepeatUntilLoops.qs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/// # Sample
+/// Repeat-Until Loops
+///
+/// # Description
+/// The repeat loop runs a block of code before evaluating a condition.
```
Comment (cesarzc) [inline samples/language/RepeatUntilLoops.qs:5]: nit: if we are referring to this as a "repeat-until loop", it should always be consistent, i.e. this sentence should be "The repeat loop runs a block of code before evaluating a condition.".

File: samples/language/WhileLoops.qs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/// # Sample
+/// While Loops
+///
+/// # Description
+/// While-loops are a basic looping looping structure in Q# for repeatedly
```
Comment (cesarzc) [inline samples/language/WhileLoops.qs:5]: nit: similar to Alex's comment, here it should probably be "While loops" instead of "While-loops".

**Review by cesarzc — APPROVED** [review]

File: samples/language/ForLoops.qs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+/// # Sample
+/// For Loops
+///
+/// # Description
+/// For-loops are a basic looping structure in Q# for looping
```
[the remark this answers is not in the document]
Comment (ScottCarda-MS) [reply samples/language/ForLoops.qs]: done

File: samples/language/RepeatUntilLoops.qs (hunk 1 above)
Comment (ScottCarda-MS) [reply samples/language/RepeatUntilLoops.qs]: done

**Pull request merged**